src/briefcase/commands/base.py

```diff
@@ -103,6 +104,26 @@ def split_passthrough(args):
         return args[:pos], args[pos + 1 :]
 
 
+def parse_config_overrides(config_overrides: list[str] | None) -> dict[str, Any]:
+    """Parse command line -C/--config option overrides.
+
+    :param config_overrides: The values passed in as configuration overrides. Each value
+        *should* be a "key=<valid TOML>" string.
+    :returns: A dictionary of app configuration keys to override and their new values.
+    :raises BriefcaseCommandError: if any of the values can't be parsed as valid TOML.
+    """
+    overrides = {}
+    if config_overrides:
+        for value in config_overrides:
+            try:
+                overrides.update(tomllib.loads(value))
+            except ValueError:
+                raise BriefcaseCommandError(
+                    f"Unable to parse configuration override {value}"
+                )
```

Comment:
I think Rich's stacktrace handles this regardless but can you raise from the `ValueError`?
```suggestion
            except ValueError as e:
                raise BriefcaseCommandError(
                    f"Unable to parse configuration override {value}"
                ) from e
```

Reply:
That should be unnecessary: exceptions raised within an except block are automatically chained unless you say `from None`.

Reply:
Might as well be explicit.